Movie/Movie/CinemaListScreen/Model/CinemaListScreenViewModel.swift

```diff
@@ -12,47 +12,58 @@ final class CinemaListScreenViewModel: CinemaListScreenViewModelProtocol {
     var onUpcomingCinemaTapHandler: ((ViewData) -> ())?
     var fetchImageHandler: ((String, Data) -> ())?
     var showErrorAlertHandler: ((String) -> Void)?
+    var showApiKeyAlertHandler: (() -> ())?
```

Comment:
```suggestion
    var showApiKeyAlertHandler: VoidHandler?
```
вынеси все в typealias